NewHaven/test/GBMapDriver.cpp

```diff
@@ -0,0 +1,69 @@
+#include <iostream>
+#include <boost/graph/graph_utility.hpp>
+#include "../src/GBMap.h"
+#include "../src/Resources.h"
+
+using namespace std;
+
+int main() {
+    GBMap *map = new GBMap();
+    int configuration = 0;
+
+    input_configuration:
+    cout << "Enter the board configuration (0 = 2 players, 1 = 3 players, 2 = 4 players): ";
+    cin >> configuration;
+
+    try {
+        map->setBoardConfig(configuration);
+    }catch(int e){
+    cout << "ERROR: " << configuration << " is not a correct option: Please select 0, 1 or 2\n";
```

Comment:
Entering an letter instead of a number isn't caught and so it throws the program into an infinite loop when it's time to input a harvest tile square #. It's probably not worth fixing for this assignment, since it's just a test driver. Just letting you know.

Reply:
Yeah I was debating whether i would go hardcore on the error catching but it didn't seem worth while for the purpose of the assignment